README.md

```diff
@@ -1,69 +1,115 @@
-# MinIO Python SDK for Amazon S3 Compatible Cloud Storage [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io)
+# MinIO Python Client SDK for Amazon S3 Compatible Cloud Storage [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io) [![Sourcegraph](https://sourcegraph.com/github.com/minio/minio-py/-/badge.svg)](https://sourcegraph.com/github.com/minio/minio-py?badge) [![Apache V2 License](https://img.shields.io/badge/license-Apache%20V2-blue.svg)](https://github.com/minio/minio-py/blob/master/LICENSE)
```

Comment:
The minio-go README includes these, added here as well. Maybe the "used by" (Sourcegraph) is incorrect? Says 0 projects.

Reply:
Removed the sourcegraph badge, left the license. If there's something more useful here, can add whatever seems helpful.